resources/views/site/blocks/collection_block.blade.php

```diff
@@ -0,0 +1,90 @@
+@php
+    use App\Helpers\GtmHelpers;
+
+    $browserArtworks = $block->browserIds('artworks');
+    $artworks = \App\Models\Api\Artwork::query()->ids($browserArtworks)->get();
+    $heading = $block->input('collection_heading');
+    $bottom_desc = $block->input('bottom_desc');
+    $primary_button_label = $block->input('primary_button_label');
+    $primary_button_link = $block->input('primary_button_link');
+    $secondary_button_label = $block->input('secondary_button_label');
+    $secondary_button_link = $block->input('secondary_button_link');
+
+    $sizes = [
+            'xsmall' => $sizes['xsmall'] ?? 2,
+            'small' => $sizes['small'] ?? 2,
+            'medium' => $sizes['medium'] ?? 3,
+            'large' => $sizes['large'] ?? 4,
+            'xlarge' => $sizes['xlarge'] ?? 4,
+    ];
+
+@endphp
+
+<div class="m-collection-block">
+    <div class="m-collection-heading">
+        @if ($heading)
+            <h2 id="{{ StringHelpers::getUtf8Slug($heading) }}" class="title f-module-title-2">{{ $heading }}</h2>
+        @endif
+        <div class="m-collection-slider-controls" data-behavior="">
+            <button class="b-drag-scroll__btn-prev btn btn--transparent f-buttons arrow-link--back f-link"><svg class="icon--arrow"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon--arrow--24"></use></svg></button>
+            <button class="b-drag-scroll__btn-next btn btn--transparent f-buttons arrow-link f-link"><svg class="icon--arrow"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon--arrow--24"></use></svg></button>
+        </div>
+    </div>
+    <div class="m-collection-artworks" data-behavior="dragScroll">
+        <div class="m-collection-row">
```

Comment:
To make it easier for editors to adjust order the artworks are divided by odds and evens so the display order of artworks will look something like:

[artwork 1] => [artwork 3]
        ||        //             ||
        v       //              v
[artwork 2]      [artwork 4]